packages/legend-application-query/src/stores/QueryBuilderFilterState.ts

```diff
@@ -407,6 +409,7 @@ export class QueryBuilderFilterState
   isRearrangingConditions = false;
   operators: QueryBuilderFilterOperator[] = [];
   private _suppressClickawayEventListener = false;
+  allowDnDProjectionToFilter = true;
 
```

Comment:
do we ever set this flag to false?

Reply:
For now, it's always set to true. Based on An's feedback last time, we'd like to provide a way to turn it on/off.

Reply:
lets write a note here. It is a bit confusing since we are not using the flag. The note should say, this flag will be leveraged when we introduce the concepts of workflows into query builder